### PR TITLE
Add support for Apache Aurora job descriptions

### PR DESCRIPTION
**Pull request by benley**

### CODE REVIEW

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -217,6 +217,15 @@ Augeas:
   tm_scope: none
   ace_mode: text
 
+Aurora:
+  type: programming
+  color: "#62BEAD"
+  group: Python
```
Comment (pchaigno) [inline lib/linguist/languages.yml]: This line means that Aurora will be counted as Python in the statistics. Is that what you want?

**Comment by pchaigno** [comment]
See [the contribution guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language) to add a language ;)

**Comment by arfon** [comment]
> See the contribution guidelines to add a language ;)

@benley - yes please take a look at these. Some samples and links to examples of usage.

**Comment by benley** [comment]
oops, I forgot about this pull request.  Thank you for the pointers, and I'll get this updated asap.

**Comment by bkeepers** [comment]
:flags: flagging as stale :flags:

**Pull request closed without merging**